assets/js/components/legacy-setup/wizard-progress-step.js

```diff
@@ -20,7 +20,6 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import React from 'react';
```

Comment:
😅

Reply:
Yeah, oddly though I still saw hook errors after removing this until the change was added to the webpack config.